tests/js/spec/views/inviteMember/inviteMember.spec.jsx

```diff
@@ -146,37 +150,28 @@ describe('CreateProject', function() {
 
       Client.addMockResponse(inviteRequest);
 
-      let wrapper;
-
-      // 👺 ⚠️ this is a hack to defeat the method auto binding so we can fully stub the method. It would not be neccessary with es6 class components and it relies on react internals so it's fragile - maxbittker
-      const index =
-        InviteMember.prototype.__reactAutoBindPairs.indexOf('redirectToMemberPage') + 1;
-
-      InviteMember.prototype.__reactAutoBindPairs[index] = () => {
```

Comment:
honestly just removed this because it was causing timeouts and I think testing the invite request mock is good enough